package-overrides/npm/redbox-react@1.1.1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "registry": "npm",
+  "main": "dist/redbox",
```

Comment:
Ok right I see. So it's loading the module and expecting it to be the full module. This is actually not an interop bug, but the way that ES modules work - they are always namespace objects when loaded unlike CommonJS. The fact that the module is loaded at all is actually pretty nice to see despite the code being designed for Node! Perhaps a simple check like `module.default || module` check within `react-transform-catch-errors` is what is needed here?